internal/rbd/nodeserver.go

```diff
@@ -1218,7 +1253,10 @@ func (ns *NodeServer) NodeGetVolumeStats(
 // connect to the Ceph cluster.
 //
 // TODO: https://github.com/container-storage-interface/spec/issues/371#issuecomment-756834471
-func blockNodeGetVolumeStats(ctx context.Context, targetPath string) (*csi.NodeGetVolumeStatsResponse, error) {
+func blockNodeGetVolumeStats(
+	ctx context.Context,
+	targetPath string,
+	volCondition *csi.VolumeCondition) (*csi.NodeGetVolumeStatsResponse, error) {
```

Comment:
This feels wrong. No need to pass `*csi.VolumeCondition` if it is not consumed. Just set the `VolumeCondition` in the function that calls `blockNodeGetVolumeStats()` instead.